### PR TITLE
Rename autolinking-ios.rb script and bring RNTester and template in line.

### PR DESCRIPTION
**Pull request by alloy**
## Summary

Make the script ready for picking in 0.62.0-rc.3.

* Rename the script to remove ambiguity. “autolinking” is a CLI feature where 3rd-party pods are automatically discovered: https://github.com/react-native-community/cli/blob/master/docs/autolinking.md
* Make the Flipper version of the pods configurable, as requested [here](https://github.com/facebook/react-native/pull/28044#discussion_r378662636).
* Make RNTester’s Podfile use the same form as the template–including enabling Flipper by moving the CPP define into the right place in the Xcode project and out of the Podfile.
* Get rid of Podfile.lock thrashing by making the default CDN spec source explicit.

## Changelog

[iOS] [Changed] - Disambiguate autolinking-ios.rb script from CLI’s “autolinking” feature and bring RNTester & template in line.

## Test Plan

RNTester builds, as does a new application.

### CODE REVIEW

File: scripts/react_native_pods.rb [hunk 1]
```diff
@@ -57,8 +57,7 @@ def use_react_native! (options={})
   end
 end
 
-def add_flipper_pods!
-  version = '~> 0.30.0'
+def add_flipper_pods!(version = '~> 0.30.2')
```
Comment (alloy) [inline scripts/react_native_pods.rb]: I’m going to rename this to `use_flipper!` to be in line with the other methods in this file.

**Comment by alloy** [comment]
Hmm, I see that I accidentally pushed this to the Facebook remote, my bad. I’ll configure my checkout to default to my fork for next time.

**Comment by TheSavior** [comment]
Can you rebase on top of master? I got some merge conflicts when importing. Looks like this file may have been changed since

**Comment by TheSavior** [comment]
Can you rebase on top of master? I got some merge conflicts when importing. Looks like one of these files may have been changed since

**Comment by alloy** [comment]
Can’t force push to this branch on the fb repo, so created a new PR from my fork https://github.com/facebook/react-native/pull/28077

**Pull request closed without merging**